components/chef-ui-library/package.json

```diff
@@ -54,6 +53,7 @@
     "mkdirp": "1.0.0",
     "node-notifier": "^10.0.0",
     "node-sass": "^6.0.1",
+    "prismjs": "^1.29.0",
```

Comment:
since prismjs is moved to dev dependecy so is it not used in any functionality in library?

Reply:
Moving it back to dependency, it is actually used in automate-ui